docs/tech/rfc/0019-240310-cesium-race-conditions.md

```diff
@@ -7,129 +7,833 @@
 
 # 0 - Summary
```

Comment:
These should be `<h2>`s instead of `<h1>`s. You should only have one `<h1>` for document as the main title.